# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/media/webrtc/libwebrtcglue/TaskQueueWrapper.cpp
# Commit: d5d11f9ac765
# Full Hash: d5d11f9ac765493245a020af66bb1c371ba3ba1a
# Author: Andreas Pehrson <apehrson@mozilla.com>
# Date: 2021-11-01 18:17:33
# Regressor Bug: 1654112
# File Overlap Count: 1
# Description:
#   Bug 1654112 - Fix static-analysis errors. r=bwc
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D108079
# ==============================================================================

diff -r 83122b946cad -r d5d11f9ac765 dom/media/webrtc/libwebrtcglue/TaskQueueWrapper.cpp
--- a/dom/media/webrtc/libwebrtcglue/TaskQueueWrapper.cpp	Thu Feb 25 17:40:18 2021 +0100
+++ b/dom/media/webrtc/libwebrtcglue/TaskQueueWrapper.cpp	Wed Mar 10 18:22:57 2021 +0100
@@ -12,7 +12,7 @@
   const UniquePtr<TaskQueueWrapper> mTaskQueue;
   MediaEventListener mShutdownListener;
   const UniquePtr<media::ShutdownBlockingTicket> mTicket;
-  MainWorker(UniquePtr<TaskQueueWrapper> aTaskQueue)
+  explicit MainWorker(UniquePtr<TaskQueueWrapper> aTaskQueue)
       : mTaskQueue(std::move(aTaskQueue)),
         mTicket(MakeUnique<media::ShutdownBlockingTicket>(
             u"MainWorker::mTicket"_ns, NS_LITERAL_STRING_FROM_CSTRING(__FILE__),